app/controllers/carts_controller.rb

```diff
@@ -3,5 +3,6 @@
 class CartsController < ApplicationController
   def show
     @cart = current_order
+    @order_total = Order.joins(:items).pluck(:price).sum
```

Comment:
Should be in the model.
Try something like `Order.joins(:items).where(id: self.id).pluck(:price).sum`
And : `@order_total = current_order.total`